docs/background/kubernetes/gardener/autoscaling.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Autoscaling allows your cluster to grow and shrink as needed.
+---
+# Autoscaling
+
+Autoscaling is the ability of your {{k8s_management_service}} shoot cluster to add and remove nodes as needed, based on the workload you deploy to the Kubernetes cluster.
+
+Autoscaling in {{brand}} is always **horizontal** rather than vertical, meaning {{k8s_management_service}} adds or removes nodes of the pre-defined machine type ([flavor](../../../reference/flavors/index.md)), rather than changing the flavor of existing nodes in-place.
+
+## Default Settings
+
+By default, {{k8s_management_service}} launches 3 initial worker nodes, and limits your cluster to a maximum of 5 nodes.
+
+You can modify these settings (*Autoscaler Min* and *Autoscaler Max*) when you [create a new shoot cluster](../../../howto/kubernetes/gardener/create-shoot-cluster.md), or at any time thereafter.
+
+## How autoscaling works
+
+Autoscaling is designed to be a "hands-off" cluster feature.
+
+Once your cluster decides its resources are insufficient to manage the current workload, it adds nodes to handle it.
+This is called **scale-out.**
+
+When the workload becomes less, it removes the extra nodes again.
+This is called **scale-in.**
+
+Scale-out is triggered if any [Pod](https://kubernetes.io/docs/concepts/workloads/pods/) on the cluster fails to be [scheduled](https://kubernetes.io/docs/concepts/scheduling-eviction/) to an existing node.
+This may be due to the fact that the node is already overloaded (meaning its `DiskPressure`, `MemoryPressure` or `PIDPressure` [node condition](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#node-conditions) is active), or a newly launched Pod is configured with a [request](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) that none of the existing nodes can meet.
```

Comment:
Nothing important, it's just that I find "due to" to be somewhat rigid.

```suggestion
This may be because the node is already overloaded (meaning its `DiskPressure`, `MemoryPressure` or `PIDPressure` [node condition](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#node-conditions) is active), or because a newly launched Pod is configured with a [request](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) that none of the existing nodes can meet.
```

Reply:
Good suggestion.